model/post.go

```diff
@@ -31,6 +31,7 @@ type Post struct {
 	Id            string          `json:"id"`
 	CreateAt      int64           `json:"create_at"`
 	UpdateAt      int64           `json:"update_at"`
+	EditAt        int64           `json:"edit_at"`
```

Comment:
Just wondering if there is a reason for adding `EditAt` and not using the existing `UpdateAt` field?